scripts/stats_compare.js

```diff
@@ -0,0 +1,89 @@
+function calculateDifference(object1, object2) {
+  const difference = {};
+
+  for (const key in object1) {
+    if (typeof object1[key] === "object") {
+      difference[key] = calculateDifference(object1[key], object2[key]);
+    } else if (typeof object1[key] === "number") {
+      difference[key] = object2[key] - object1[key];
+    }
+  }
+
+  return difference;
+}
```

Comment:
This function looks like it recursively maps each value in a pair of objects to the difference between the objects’ numeric values. Please add a comment explaining as much.